sysutils/smart/Makefile

```
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		smart
-PLUGIN_VERSION=		2.2
+PLUGIN_VERSION=		2.3
```

let's only bump revision here

```suggestion
PLUGIN_VERSION=		2.2
PLUGIN_REVISION=		1
```